.github/workflows/ci-it.yaml

```diff
@@ -38,11 +38,26 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 8
+
+      - name: Setup reviewdog
```

Comment:
Have you requested open `reviewdog` on GitHub?

Reply:
no need